app/src/test/java/App2/AppTest.java

```diff
@@ -3,12 +3,15 @@
  */
 package App2;
 
-import org.junit.jupiter.api.Test;
-import static org.junit.jupiter.api.Assertions.*;
+//import org.junit.jupiter.api.Test;
```

Comment:
Why do we remove the test and assertions imports?

Reply:
Assertions isn't used. Removing test import was probably an accident :s

Reply:
also the test isnt used